### DISCUSSION TITLE
vdhcoapp is infected with malware Avast File Shield alert

### DISCUSSION THREAD

**Discussion by vrmex** [aclap-dev/video-downloadhelper#103] [Help and Q&A] [answered]
### Your question

I just got an Avast File Shield alert: vdhcoapp is infected with malware: MacOS:Agent-AIU [Tool] move to Quarantine. 
What shold I do? 
Why Is your software now infected with malware?

**Comment by paulrouget** [comment, marked as answer]
> Why Is your software now infected with malware?

It's not. Anybody can verify this, code is Open Source:: https://github.com/aclap-dev/vdhcoapp/ - Avast is flagging the CoApp wrongly.
